### PR TITLE
Wizard recipe: UDUNITS-v2.2.28

### PR DESCRIPTION
**Pull request by simonbyrne**
This pull request contains a new build recipe I built using the BinaryBuilder.jl wizard:

* Package name: UDUNITS
* Version: v2.2.28

@staticfloat please review and merge.

### CODE REVIEW

**Comment by simonbyrne** [comment]
I'm not sure about the naming here (in particular the capitalization):
https://www.unidata.ucar.edu/software/udunits/udunits-2.2.28/udunits2.html

cc: @jakebolewski

**Comment by giordano** [comment]
I think you're seeing #3949 and you have to manually set
```
export CPPFLAGS="-I${includedir}"
```
before running configure

File: U/UDUNITS/build_tarballs.jl [hunk 1]
```diff
@@ -0,0 +1,38 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "UDUNITS"
+version = v"2.2.28"
+
+# Collection of sources required to complete build
+sources = [
+    ArchiveSource("https://artifacts.unidata.ucar.edu/repository/downloads-udunits/udunits-$(version).tar.gz", "590baec83161a3fd62c00efa66f6113cec8a7c461e3f61a5182167e0cc5d579e")
```
Comment (giordano) [inline U/UDUNITS/build_tarballs.jl:10]: https://github.com/JuliaPackaging/BinaryBuilder.jl/pull/1148 works, yes!

**Comment by giordano** [comment]
Capitalization of the name is fine

**Comment by giordano** [comment]
My educated guess for powerpc is https://docs.binarybuilder.org/stable/troubleshooting/#Shared-library-not-built, I don't know what's up with freebsd though

**Comment by simonbyrne** [comment]
@giordano any suggestions on what to do about:
https://github.com/Unidata/UDUNITS-2/blob/c83da987387db1174cd2266b73dd5dd556f4476b/lib/xml.c#L2140
on BSD?

**Comment by simonbyrne** [comment]
Also not sure what the issue with PPC is (it looks like it isn't building a shared library?). @vchuravy any ideas?

**Comment by giordano** [comment]
The `__FreeBSD__` macro should be defined on freebsd

**Comment by jakebolewski** [comment]
> @giordano any suggestions on what to do about: https://github.com/Unidata/UDUNITS-2/blob/c83da987387db1174cd2266b73dd5dd556f4476b/lib/xml.c#L2140 on BSD?

I would try `if not defined WIN32` for that pre-processor branch

**Comment by simonbyrne** [comment]
Okay, now I'm stuck. All the builds are failing with:
```
[01:06:40] WARNING: 'makeinfo' is missing on your system.
[01:06:40]          You should only need it if you modified a '.texi' file, or
[01:06:40]          any other file indirectly affecting the aspect of the manual.
[01:06:40]          You might want to install the Texinfo package:
[01:06:40]          <http://www.gnu.org/software/texinfo/>
[01:06:40]          The spurious makeinfo call might also be the consequence of
[01:06:40]          using a buggy 'make' (AIX, DU, IRIX), in which case you might
[01:06:40]          want to install GNU make:
[01:06:40]          <http://www.gnu.org/software/make/>
```

**Comment by jakebolewski** [comment]
Install the texinfo alpine pkg?  I would imagine there is a flag for disabling the doc building

**Comment by simonbyrne** [comment]
What's weird is that it only appeared after running autoreconf.

**Comment by simonbyrne** [comment]
Okay switching to cmake fixed it.

**Comment by giordano** [comment]
CMake to the rescue, who'd have thought that? :smirk:

**Pull request merged**